R/lazy.R

```diff
@@ -116,7 +116,7 @@ print.tbl_connect <- function(x, ..., n = NULL) {
 
 #' @export
 as.data.frame.tbl_connect <- function(x, row.names = NULL, optional = NULL, ..., n = Inf) {
-  as.data.frame(collect(x, n = n))
+  as.data.frame(dplyr::collect(x, n = n))
 }
 
 op_base_connect <- function(x, vars) {
```

Comment:
I'm not familiar enough w/ this space, so started w/ only the directly-conflicting methods! :)